extensions/migadu/src/utils/api.ts

```diff
@@ -28,8 +28,17 @@ const headers = {
   "Content-Type": "application/json",
 };
 
-const callApi = async <T>(endpoint: string, method: APIMethod, body?: BodyRequest) => {
-  await showToast(Toast.Style.Animated, "Processing...");
+export type ApiToastVerbosity = "verbose" | "error";
```

Comment:
I added this because the fetch of all mailboxes was triggering a "success" toast the moment the user opens the command. IMO, this isn't useful information, hence this refactoring. Let me know if you disagree or want this implemented some other way.

Reply:
That's fine. This was one of the earlier extensions I authored so it doesn't have a lot of best practices - we might re-write this function and phase out said Toasts eventually.